src/app/static/src/components/parameters/EditPhases.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div>
+    <modal class="phase-modal" :open="open">
+      <h3>Edit {{paramGroup && paramGroup.label}}</h3>
+      <p>Click on a Phase to drag it to a new start date.</p>
+        <div class="phase-editor" @mouseup="mouseUp">
+          <div class="phases-container">
+            <div ref="rail" class="rail">
+              <div v-for="(value, index) in sliderValues" :id="`slider-${index}`"
+                   :key="index"
+                   role="slider"
+                   :tabindex="index"
+                   :style="`left:${sliderPosition(value)}%;z-index:${value.value.zIndex};`"
```

Comment:
Perhaps [object syntax](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1) could make this slightly clearer, if it's possible to use it in this instance